app/lib/services/restApi.registerPushToken.test.ts

```
@@ -0,0 +1,133 @@
+import { store as reduxStore } from '../store/auxStore';
+import sdk from './sdk';
+import * as restApi from './restApi';
```

ESLint

[error] There should be no empty line within import group

(import/order)